app/assets/javascripts/admin/modules/taxonomy_tree_checkboxes.js

```diff
@@ -56,8 +81,10 @@
         and the content could be shown to users seeking any of them.
         */
         if (checked) {
+          checkboxTrackClick("checkboxClickedOn", options);
```

Comment:
@MatMoore this only triggers when the user clicks the actual checkbox, the ancestors will not be triggering this, checking the ancestors is responsibility of `checkAncestors(this)` function

Here I've clicked on "Maths" and "GCSE subject content and requirements"
![screen shot 2017-02-22 at 11 05 30](https://cloud.githubusercontent.com/assets/136777/23208928/e223893c-f8ee-11e6-9764-cdc86a098062.png)


Here I've unchecked "GCSE subject content and requirements"

![screen shot 2017-02-22 at 11 06 06](https://cloud.githubusercontent.com/assets/136777/23208939/edb0a604-f8ee-11e6-8deb-11caaeba2c15.png)


You can see from the screenshots that the event is only triggered once per click.
I hope that answers your question.